### ISSUE TITLE
setting clock: timezone applied twice

### ISSUE THREAD

**Issue by chrand** [zwave-js/zwave-js-ui#3329]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

9.0.3.c37f5a4

### ZwaveJS version

12.0.2

### Describe the bug

Z-Wave JS sends clock to nodes, applying its own timezone, also to devices which already have a timezone setting. Therefore, nodes will show time = UTC + timezone_controler + timezone_node (one timezone too much).

### To Reproduce

How to reproduce: in Z-WaveJS UI -> [node] -> advanced -> set clock, I push clock from server to nodes. Operation completes successfully, and nodes get updated. However, device shows wrong clock, off by yet-another timezone setting. In my case: Z-WaveJS UI is deployed on docker, setting timezone (UTC+1). Nodes are thermostats, with a configured setting of the same timezone (UTC+1), among their node configuration.

### Expected behavior

Server could optionally send clock, without timezone correction (i.e. in UTC). Having nodes in UTC would not be the best workaround, because of automation (programmed on node).

### Additional context

Controller is an usb, Silicon Labs, Aeotec 7-Stick, f/w. 7.19.1

**Comment by robertsLando** [comment]
> Z-WaveJS UI is deployed on docker, setting timezone (UTC+1). Nodes are thermostats, with a configured setting of the same timezone (UTC+1), among their node configuration.

Are you using the TZ env var on your docker-compose file?

**Comment by chrand** [comment]
> Are you using the TZ env var on your docker-compose file?

Yes, the docker-compose is:
`zwave-js-ui:
  image: zwavejs/zwave-js-ui:latest
  ...
  environment:
    - TZ=...`

**Comment by robertsLando** [comment]
@AlCalzone any clue?

Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field). Do that so it include a set clock call

**Comment by AlCalzone** [comment]
Yeah I'll need to see the driver log of a re-interview. Setting time is complex as there are about 4 ways to do that - some support timezone, some don't. And it's not impossible that the device has a bug or there's some incorrect assumption involved.
